tests/gocase/integration/slotmigrate/slotmigrate_test.go

```diff
@@ -413,22 +413,18 @@ func TestSlotMigrateSync(t *testing.T) {
 
 	t.Run("MIGRATE - Migrate sync with (or without) all kinds of timeouts", func(t *testing.T) {
 		slot++
-		require.Equal(t, "OK", rdb0.Do(ctx, "clusterx", "migrate", slot, id1, "sync").Val())
-
-		slot++
-		require.Equal(t, "OK", rdb0.Do(ctx, "clusterx", "migrate", slot, id1, "sync", -1).Val())
-
-		slot++
-		require.Equal(t, "OK", rdb0.Do(ctx, "clusterx", "migrate", slot, id1, "sync", 0).Val())
-
-		slot++
-		require.Equal(t, "OK", rdb0.Do(ctx, "clusterx", "migrate", slot, id1, "sync", 10).Val())
-
-		slot++
-		require.Equal(t, "OK", rdb0.Do(ctx, "clusterx", "migrate", slot, id1, "sync", 0.5).Val())
-
-		slot++
-		require.Equal(t, "OK", rdb0.Do(ctx, "clusterx", "migrate", slot, id1, "sync", -3.14).Val())
+		// migrate sync without explicitly specify the timeout
+		result := rdb0.Do(ctx, "clusterx", "migrate", slot, id1, "sync")
+		require.NoError(t, result.Err())
+		require.Equal(t, "OK", result.Val())
+
+		timeouts := []interface{}{-1, 0, 20, 10.5, -3.14}
```

Comment:
Could we use `[]float64` here instead of `[]interface{}`?
If the timeout is a negative value, should it be an error?

Reply:
I see that right now timeout < 0 doesn't produce an error. Should we introduce an error in this case? Or just treat it as an absence of a timeou?

Reply:
I will append a commit to resolve above comments.

Reply:
@torwig @git-hulk The original reason to support float timeout is that in the go tests I want to "deliberately let the sync migration timeout" (see [here](https://github.com/apache/incubator-kvrocks/blob/unstable/tests/gocase/integration/slotmigrate/slotmigrate_test.go#L440)). And the minimal integer value of "1" is too large and cannot ensure the timeout happen...

Also I think supporting float timeout is not harmful but provides more precise control to the migration. What do you think?

As for the negative value, returning an error when the server receives an negative value seems acceptable to me. No problem.

Reply:
@ellutionist Timeout is in seconds as far as I see so I don't think that making the timeout less than 1 second is reasonable (it's about migrating slot with data from one node to another via network).

Reply:
@ellutionist Another concern is float timeout may be inconsistent with other commands.